### PR TITLE
Improve understandability of kubelet's cpu accumulator code

### PR DESCRIPTION
**Pull request by matte21**
#### What type of PR is this?
/kind documentation
/kind cleanup

#### What this PR does / why we need it:

The cpu accumulator logic (that select CPUs for containers) has some non-obvious code (as someone reading it for the first time it took me a long time to understand it).
This PR adds some comments to make that code easier to understand for new contributors.

There are also some minor renames of functions and variables to improve readability.

#### Special notes for your reviewer:
The comments are very verbose. Perhaps we can streamline them.

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

### CODE REVIEW

**Comment by ffromani** [comment]
/ok-to-test
/triage accepted
/priority backlog

code documentation is fine and everything else looks like trivial renames.
I'll have a proper review later on

**Review by swatisehgal — COMMENTED** [review]
/lgtm

**Review by Hemen0 — APPROVED** [review]

File: pkg/kubelet/cm/cpumanager/cpu_assignment.go
Comment (rajibmitra) [inline pkg/kubelet/cm/cpumanager/cpu_assignment.go:1]: /lgtm

**Review by rajibmitra — APPROVED** [review]

**Comment by matte21** [comment]
@rajibmitra @swatisehgal @ffromani can someone add this to milestone 1.29 so that it can be merged?

**Comment by matte21** [comment]
@rajibmitra @swatisehgal @ffromani can someone add this to milestone 1.29 so that it can be merged?

**Comment by matte21** [comment]
cc @rajibmitra @swatisehgal @ffromani

**Comment by ffromani** [comment]
> cc @rajibmitra @swatisehgal @ffromani

Hi! the change already got LGTM'd. Only leads can add or remove to the milestone. CC @mrunalp @SergeyKanzhelev .
It's probably easier to wayt for the 1.30 to open and this change should merge automatically.

**Comment by matte21** [comment]
Thanks for the infromation!

**Review by mrunalp — APPROVED** [review]

**Pull request merged**